test/verify/files/mock-insights

```diff
@@ -52,13 +52,14 @@ class handler(BaseHTTPRequestHandler):
         m = self.match("/r/insights/v1/systems/([^/]+)")
         if m:
             machine_id = m[1]
-            if machine_id not in systems:
-                self.send_response(404)
-                self.end_headers()
-                return
-            self.send_response(200)
+            for system in systems.values():
+                if system["machine_id"] == machine_id:
```

Comment:
Ah, so `systems` is not indexed by machine ID any more but by "inventory ID" like "123-nice-id". So many IDs! :wink:

Reply:
well yes, Inventory has lots of IDs: the ID of a system, the machine ID, the Insights ID, etc...